modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/reconstructioncorrection/v0.1.0.json

```diff
@@ -0,0 +1,43 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/morphology/reconstructioncorrection/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:ReconstructionCorrectionShape",
+      "@type": "sh:NodeShape",
+      "label": "Reconstruction correction activity description.",
+      "targetClass": "nsg:ReconstructionCorrection",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/shapes/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "class": "nsg:ReconstructedWholeBrainCell",
```

Comment:
This class doesn't match what is in the prov template below. A ReconstructionCorrection takes an instance of nsg:ReconstructedCell and generates a revision of it which is an instance nsg:ReconstructedCell. 
Can this schema be aligned with the prov template ?